src/StoreApi/Routes/Products.php

```diff
@@ -309,10 +309,14 @@ public function get_collection_params() {
 
 		$params['stock_status'] = array(
 			'description'       => __( 'Limit result set to products with specified stock status.', 'woo-gutenberg-products-block' ),
-			'type'              => 'string',
-			'enum'              => array_keys( wc_get_product_stock_status_options() ),
+			'type'              => 'array',
+			'items'             => array(
+				'type' => 'string',
+			),
+			'default'           => [],
 			'sanitize_callback' => 'sanitize_text_field',
```

Comment:
I think `sanitize_callback` needs to be inside `items`. Otherwise, it was not working for me. :confused: